src/FSharp.Data.GraphQL.Client/FSharp.Data.GraphQL.Client.fsproj

```diff
@@ -11,6 +11,11 @@
     <Description>FSharp implementation of Facebook GraphQL query language (Client)</Description>
   </PropertyGroup>
 
+  <ItemGroup>
+    <None Include="..\..\icon.svg" Pack="true" PackagePath="" Visible="False" />
```

Comment:
Do we need that icon at all? I thought that NuGet will add it by default. I expected that we will have another icon later.
Is it your intent?

Reply:
@xperiandri yes, I saw this entry in one of the fsproj files, and I thought it was already an intent of the team, although the icon file itself was missing. We can certainly remove it at the moment.

Reply:
I've add some similar stuff to the common `Directory.Build.props`. Check that